composer.json

```diff
@@ -32,14 +32,13 @@
 		"wp-coding-standards/wpcs": "2.1.1",
 		"sirbrillig/phpcs-variable-analysis": "2.7.0",
 		"phpcompatibility/phpcompatibility-wp": "2.1.0",
-		"sirbrillig/phpcs-changed": "1.0.0"
+		"sirbrillig/phpcs-changed": "2.2.7-beta-1@dev"
```

Comment:
do we want to pin this to a beta version?

Reply:
Nevermind I see in your PR description that it's necessary and intended

Reply:
Assuming this all works fine, he'll deploy this as stable and we will update accordingly.